examples/plot_intersectional_bias.py

```diff
@@ -0,0 +1,1327 @@
+# respect to
+# the two diagnoses, in reality, Schizophrenia is much less common than Affective Disorder. If
+# collected naturally based on different patients seen at a health system, our classes would be
+# very imbalanced.
+#
+# Fairness Assessment
+# ===================
+#
+# While seemingly naive, many ML algorithms deployed in the past have indeed failed to properly
+# address fairness. Perhaps the most infamous example of this is the COMPAS algorithm
+# :footcite:`angwin2016machine` for prediction criminal recidivism, where Black defendants were
+# found to have higher risk scores, a higher false positive rate, and a lower false negative rate
+# compared to white defendants. In other applications of ML, the lack of fairness concerns proves
+# pervasive. An audit of three facial recognition software from IBM and Microsoft found the error
+# rate for darker-skinned females to be 34% higher than for lighter-skinned males
+# :footcite:`buolamwini2018gender`. When translating into a gendered language, Google Translate
+# skews towards masculine translations for words like “strong” or “doctor,” while skewing feminine
+# for words like “beautiful” or “nurse.” :footcite:`kuczmarski2018reducing`
+#
+# These are clear problems which we aim to avoid by performing a proper fairness assessment. Let's
+# do that now.
+#
+# Quantifying fairness
+# ~~~~~~~~~~~~~~~~~~~~
+#
+# Many fairness metrics are calculated from basic performance metrics, like sensitivity,
+# specificity, TPR, FPR, etc. There are many performance metrics available, so we need to select
+# one that is most relevant to our task.
+#
+# One common (and arguably most successful to date) application of ML in healthcare is in the
+# diagnosis of tumours from medical images (like chest x-rays) as being cancerous or non-cancerous.
+#
+# **In this task, which performance metrics are most relevant?**
+#
+# In this case, **false negative predictions** would probably be most relevant, because when
+# classifying tumours as being cancerous or non-cancerous, we would prefer to mistake a benign
+# tumour as being cancerous, as compared to the other way around. This is called a false positive
+# (or misdiagnosis), and it’s better than a false negative (or underdiagnosis) in this specific
+# circumstance.
+#
+# Why? Because a false negative would result in a false perception that the patient does not have
+# cancer, which could delay potentially life-saving treatment.
+#
+# There is a study suggesting that, when diagnosing the nature of tumours from medical images,
+# false negative rates are higher for disadvantaged or underserved groups defined by intersecting
+# features (e.g., low-income, Black women). :footcite:`seyyedkalantari2021underdiagnosis` This is an excellent,
+# real-world example of how the application of ML in healthcare can *amplify harms*, if deployed
+# without concern for fairness.
+#
+# *Which performance metric is most important in our case?*
```

Comment:
Something like the Fairness Decision Tree provided by Aequitas may be easier for a layperson to understand how to make this choice. http://www.datasciencepublicpolicy.org/our-work/tools-guides/aequitas/

Reply:
Perhaps we can reference it in the above paragraph?

> Many fairness metrics are calculated from basic performance metrics, like sensitivity, specificity, TPR, FPR, etc. There are many performance metrics available, so we need to select one that is most relevant to our task: decision trees like [this](http://www.datasciencepublicpolicy.org/our-work/tools-guides/aequitas/) one provided by Aequitas provide an easy way to understand the use cases for different metrics.

> As an example, one common (and arguably most successful to date) application of ML in healthcare is in the diagnosis of tumours from medical images (like chest x-rays) as being cancerous or non-cancerous.

Reply:
I've added the suggestion by @yifansmart